src/constants/index.js

```diff
@@ -1 +1 @@
-export const IMAGE_FALSE_AVP_TRUE = true;
+export const IMAGE_FALSE_AVP_TRUE = false;
```

Comment:
is this documented somewhere?

Reply:
https://github.com/DynamoDS/DynamoDictionary#toggle-avp--example-images

@ScottAMitchell I don't think you want to commit this change, just locally for testing the loading of images

Reply:
@alfarok yep that makes sense